src/main.rs

```diff
@@ -207,6 +209,10 @@ fn main() {
     // (sending telemetry and downloading genesis)
     openssl_probe::init_ssl_cert_env_vars();
 
+    // This is a sanity check. Indexer should fail .env file with credentials is missing
+    // We prefer to fail as soon as possible to avoid heavy running for nothing
+    models::get_database_credentials();
```

Comment:
FYI, from the call sight it is not immediately obvious why it should fail. We ask to get the credentials and don't check anything, and discard the result completely. I know we panic there, but it is not ideal. Still, it works.

I propose we add the `database_url` to the clap structure and configure it with `DATABASE_URL` env variable: https://docs.rs/structopt/0.3.21/structopt/?search=#environment-variable-fallback, this way we will get a nice error message from clap, and we don't need to have this hacky function that goes around the CLI arguments

Reply:
I've extracted it to separate issue #97 to deal with it later.